airesources/Haskell/MyBot.hs

```diff
@@ -0,0 +1,44 @@
+import Hlt.Entity
+import Hlt.GameMap
+import Hlt.Navigation
+import Hlt.Networking
+
+-- | Define the name of our bot.
+botName :: String
+botName = "AdamBot"
+
+-- | Log to a file
+info :: String -> IO ()
+info s = appendFile (botName ++ ".log") (s ++ "\n")
```

Comment:
Can you add player id to the file name? Otherwise there will be an issue if two instances of the same bot are playing, right? Other kits do <id>_<botname>.log